### ISSUE TITLE
Jenkins job for Pre-Ingest tool

### ISSUE THREAD

**Issue by wkollernhm** [gbhl/bhl-europe#346]
Setup a Jenkins job for fetching the current Pre-Ingest tool code from github and deploy it on our integration server.

Code: https://github.com/bhle/bhle/tree/master/pre-ingest/ingest
Database-Dump: https://github.com/bhle/bhle/blob/master/pre-ingest/ingest/docs/database/create_database.sql

At the same time change our /ingest URL to /preingest !

@mehrrath 
@lobajuluwa 
@chris-sleep 
@ZhengLIAtos

**Comment by wkollernhm** [comment]
Please also keep in mind, that for queueing a cron-job has to be set-up which runs https://github.com/bhle/bhle/blob/master/pre-ingest/ingest/queuing/run_preingest_jobs.sh every 10 minutes!

(it is currently already set-up, but if you change the URL you also have to update the cron-job)

**Comment by zhengl** [comment]
Done.

Please refer to http://bhle-dev-1.nhm.ac.uk/job/pre-ingest/

This job replaces the sensitive information in config files and dump database. But it does not update the location of cron job, which is not a frequent update.

**Comment by zhengl** [comment]
Please update Pre-ingest Tool to Github.

@mehrrath 
@wkollernhm

**Comment by wkollernhm** [comment]
I've just updated the code on github to contain the latest from /var/www/ingest/ - in addition I fixed a small typo in the Jenkins Job (chown instead of chmod)!

In addition I removed the var/www/ingest/ folder to avoid any future updates to it!

**Comment by zhengl** [comment]
Thanks!

2012/2/27 wkollernhm <
reply@reply.github.com

**Comment by mindcatch-com** [comment]
ok

**Comment by wkollernhm** [comment]
The FSTrigger plugin always triggered itself since we were changing the config file during build, this means the deployment process was deploying the tool way to often.

I installed the "Files Found Trigger" Plugin which monitors for the config_dev.php file, if it exists it triggers a build. In the build-script I have added a command to delete the config_dev.php file which again disables building until the next pull from github.

@ZhengLIAtos 
@leenamba 
@akohlbecker 
@chris-sleep
